README.md

```diff
@@ -273,10 +273,10 @@ Arguments:
 
 Display the list of currently remembered directories. Returns an array of paths in the stack, or a single path if +N or -N was specified.
 
-See also: pushd, popd
+See also: `pushd`, `popd`
 
 
-### echo([options,] string [, string ...])
+### `echo([options,] string [, string ...])`
```

Comment:
I think it looks a bit better with the headers as-is. Were there any formatting bugs I'm overlooking?

Reply:
No.  I just prefer the monospace font for any documentation that is referencing a command/option/option-value.  

But I can omit this when I redo the Pull Request.  :)